# HG changeset patch
# Parent 2599cf4f33cf5b2b588346125060ce0947c04796
# User sbr
# Date 1347552011 -7200

Fix: Two INVALID_STRING_ID are missing in _station_load_types.

diff -r 2599cf4f33cf src/order_gui.cpp
--- a/src/order_gui.cpp	Thu Sep 13 17:56:12 2012 +0200
+++ b/src/order_gui.cpp	Sun Sep 16 11:46:26 2012 +0200
@@ -60,6 +60,7 @@
 			INVALID_STRING_ID,
 			INVALID_STRING_ID,
 			INVALID_STRING_ID,
+			INVALID_STRING_ID,
 		}, {
 			STR_ORDER_NO_UNLOAD,
 			INVALID_STRING_ID,
@@ -93,6 +94,7 @@
 			INVALID_STRING_ID,
 			INVALID_STRING_ID,
 			INVALID_STRING_ID,
+			INVALID_STRING_ID,
 		}, {
 			STR_ORDER_NO_UNLOAD_REFIT,
 			INVALID_STRING_ID,
